iOSClient/Menu/NCMenu.swift

```diff
@@ -91,13 +91,6 @@ class NCMenu: UITableViewController {
     override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         return 60
     }
-
-    // MARK: - Accessibility
-
-    open override func accessibilityPerformEscape() -> Bool {
-        dismiss(animated: true)
-        return true
-    }
```

Comment:
@marinofaggiana can you confirm that this was only used for dismissing when tapping outside the menu?

Reply:
Sorry, I don't have write this code, you have to test it by yourself

Reply:
I always test manually. But I just wasn't sure if it's needed for anything else. I suppose it's ok then.